Bottom: 13342c612aa9b895f182e6d8183b2480184b6e84
Top:    e4408e3ac9afe19fbfcc20cd71152da73e3e41c1
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-13 11:32:41 +0800

Refresh of xdp1_user-support-ifindex-for

---

diff --git a/samples/bpf/xdp1_user.c b/samples/bpf/xdp1_user.c
index 604eb72..caf3f7d 100644
--- a/samples/bpf/xdp1_user.c
+++ b/samples/bpf/xdp1_user.c
@@ -63,8 +63,9 @@ static void usage(const char *prog)
 	fprintf(stderr,
 		"usage: %s [OPTS] IFINDEX\n\n"
 		"OPTS:\n"
-		"    -S    use skb-mode\n"
-		"    -N    enforce native mode\n",
+		"    -S           use skb-mode\n"
+		"    -N           enforce native mode\n"
+		"    -D ifindex   offload mode\n",
 		prog);
 }
 
@@ -74,7 +75,7 @@ int main(int argc, char **argv)
 	struct bpf_prog_load_attr prog_load_attr = {
 		.prog_type	= BPF_PROG_TYPE_XDP,
 	};
-	const char *optstr = "SN";
+	const char *optstr = "SND";
 	int prog_fd, map_fd, opt;
 	struct bpf_object *obj;
 	struct bpf_map *map;
